packages/ui/src/theme/tokens.ts

```diff
@@ -0,0 +1,6154 @@
+/**
+ * Do not edit directly
+ * Generated on Tue, 07 Sep 2021 00:13:20 GMT
```

Comment:
Because this is a generated file, should we add this to gitignore? Or if we do want to check this in, we should probably change this [file header](https://amzn.github.io/style-dictionary/#/version_3?id=custom-file-headers) comment to not have a generated datetime string. 

In the file configuration object you can pass a fileHeader function:

```js
        options: {
          fileHeader: () => {
            return [
              `Do not edit directly`
            ]
          }
```

Reply:
Sounds great! I will make the update to ignore this file